### PR TITLE
Apb 7238 [RW] add cbc to accordion (behind FF)

### PR DESCRIPTION
**Pull request by raw1697**

### CODE REVIEW

File: app/uk/gov/hmrc/agentservicesaccount/views/pages/dashboard/tax_services_accordion.scala.html [hunk 1]
```diff
@@ -135,6 +135,15 @@
 
 
 }
+    @CounrtyByCountryAccordionItem = {
+
+            @h4("asa.section3.col1.h3")
+            @p(msgs("asa.cbc-section.col1.link1.p", appConfig.agentInvitationsFrontendClientTypeUrl))
+
+            @h4("Manage country-by-country reports")
```
Comment (geoffreywatson) [inline app/uk/gov/hmrc/agentservicesaccount/views/pages/dashboard/tax_services_accordion.scala.html:143]: doesn't this need to be a message?

File: app/uk/gov/hmrc/agentservicesaccount/views/pages/dashboard/tax_services_accordion.scala.html [hunk 2]
```diff
@@ -135,6 +135,15 @@
...
+            @p(html = Some(a(href = "https://cbcr-prototype.herokuapp.com/v3/report/agent/what-do-you-want-to-do-next",
```
Comment (geoffreywatson) [inline app/uk/gov/hmrc/agentservicesaccount/views/pages/dashboard/tax_services_accordion.scala.html:144]: no links to prototypes!

File: app/uk/gov/hmrc/agentservicesaccount/views/pages/dashboard/tax_services_accordion.scala.html [hunk 3]
```diff
@@ -172,6 +181,16 @@
             content = HtmlContent(capitalGainsAccordionItem)
         )
     )
+
+    if(appConfig.enableCbc) {
+        items.append(
+            Section(
+            headingContent = Text(msgs("Country-by-country reports")),
```
Comment (geoffreywatson) [inline app/uk/gov/hmrc/agentservicesaccount/views/pages/dashboard/tax_services_accordion.scala.html:188]: need a properly formatted message key here

File: conf/application.conf [hunk 4]
```diff
@@ -169,8 +169,6 @@ auditing {
   enabled=true
   consumer {
```
Comment (geoffreywatson) [inline conf/application.conf:170]: you can remove the whole consumer section too as there's nothing in it now

File: app/uk/gov/hmrc/agentservicesaccount/views/main.scala.html [hunk 5]
```diff
@@ -93,7 +94,7 @@
         additionalHeadBlock = Some(head(hasTimeout)),
         additionalScriptsBlock = Some(scripts),
         beforeContentBlock = Some(beforeContent),
-        mainContentLayout = if(fullWidth) Some(fullWidthMainContent(_)) else None
+        mainContentLayout = if(fullWidth) Some(twoThirdsLayout(_)) else None
```
Comment (CarolineS-QA) [inline app/uk/gov/hmrc/agentservicesaccount/views/main.scala.html:97]: this should be `if(fullWidth) Some(fullWidthMainContent(_)) else Some(twoThirdsLayout(_))`

**Review by CarolineS-QA — APPROVED** [review]

**Pull request merged**